src/config.rs

```diff
@@ -3123,7 +3147,8 @@ fn to_toml_encode(change: HashMap<String, String>) -> CfgResult<HashMap<String,
         }
     }
     let mut dst = HashMap::new();
-    for (name, value) in change {
+    for (mut name, value) in change {
+        name = serde_to_online_config(name);
         let fields: Vec<_> = name
```

Comment:
```suggestion
        let online_config_name = serde_to_online_config(name);
        let fields: Vec<_> = online_config_name
```
We should use the serde format to write the toml file, also could you add some test cases of the pool size config to the test `test_to_toml_encode`?